src/styles/index.scss

```diff
@@ -1,3 +1,56 @@
+@import '/src/styles/variables';
+
 body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($size-day * 7) + ($size-gap * 6);
```

Comment:
You can use `calc()` CSS function to calculate the width. This makes the code more readable and easier to understand.